### ISSUE TITLE
Support firefox

### ISSUE THREAD

**Issue by algesten** [algesten/str0m#12]
Getting (clear) error messages in the console for Firefox. This is probably a good way to work through issues that might affect other browsers too.

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/227204/209461560-4ee0b813-69f4-4b86-880a-78a635ea9b73.png">

**Comment by algesten** [comment]
One fixed problem is that the o= contains an unsigned 64 bit session id, and firefox expects this to be capped at 2^62 - 1 (https://bugzilla.mozilla.org/show_bug.cgi?id=861895)

**Comment by algesten** [comment]
Another problem is that firefox refuses to talk webrtc to 127.0.0.1 – this is correct according to spec, but Chrome/Safari seems to be quite relaxed about it.

To fix this, we bind UDP to  0.0.0.0:0 and try to figure out some local host IP we can use for SDP purposes. I'm not thrilled with hardcoding interface names I'm hoping works in the chat example. Is there a better way? https://github.com/algesten/str0m/blob/906f115ae5d8e9a70d5e061c76db0d8c7453174b/str0m/examples/chat.rs#L120-L138

**Comment by algesten** [comment]
Next problem is strange. With one browser tab towards the chat example server, firefox seems to work fine. However if I open a second tab, I get:

```
WARN chat: Client failed: Dtls(Io(Custom { kind: InvalidData, error: Error { code: ErrorCode(1), cause: Some(Ssl(ErrorStack([Error { code: 336602130, library: "SSL routines", function: "dtls1_read_bytes", reason: "sslv3 alert bad certificate", file: "ssl/record/rec_layer_d1.c", line: 615, data: "SSL alert number 42" }]))) } }))
```

**Comment by algesten** [comment]
This seems like the problem exact problem we're seeing. https://github.com/versatica/mediasoup/issues/127#issuecomment-474460153

**Comment by algesten** [comment]
Fixed in ecb742b
